docs/reference/context/preparation-process.md

```diff
@@ -12,7 +12,7 @@ Holds the information about a [Preparation Process](../connection/ssh.md#prepara
 
 Each preparation process has its own unique ID (like [`pull-image`](../environment/docker.md#preparationProcess-pull-image) of the [docker environment](../environment/docker.md)).
 
-Please refer the documentation of the supported [environments](../environment/index.md) for their preparation process kinds.
+For information about process types, see the documentation for the supported [environments](../environment/index.md).
```

Comment:
The term preparation kinds needs do be preserved, because this is a technical term in this document.

Reply:
@CiaoOtway any update on this?